stories/components/alert/alert.handlebars

```diff
@@ -0,0 +1,12 @@
+<div class="alert {{color}}" role="alert">
+    {{# if dismissable}}<button type="button" class="alert__button" data-dismiss="alert" aria-hidden="true">X</button>{{/if}}
```

Comment:
This is flagging an accessibility error because `aria-hidden` is applied to the button element, so it's hiding the whole button from screen readers. I think the solution here is to follow the same pattern that we use for modals:

```
<button class="modal__header-button" aria-label="Close">
      <span class="material-icon" aria-hidden="true">close</span>
</button>